diff --git a/node_modules/@supersoniks/concorde/src/core/components/functional/queue/queue.ts b/node_modules/@supersoniks/concorde/src/core/components/functional/queue/queue.ts
index 511810a..f2066bc 100644
--- a/node_modules/@supersoniks/concorde/src/core/components/functional/queue/queue.ts
+++ b/node_modules/@supersoniks/concorde/src/core/components/functional/queue/queue.ts
@@ -35,6 +35,7 @@ export default class Queue extends Subscriber(LitElement, {} as QueueProps) {
   @property({ type: Function }) items: ListItems | null = null;
   @property({ type: Function }) noItems: ListItems | null = null;
   @property({ type: Function }) skeleton: ListItems | null = null;
+  @property({ type: Function }) separator: (() => unknown) | null = null;
   lastRequestTime = 0;
   key = "";
 
@@ -371,6 +372,7 @@ export default class Queue extends Subscriber(LitElement, {} as QueueProps) {
             .items=${this.items}
             .noItems=${index == 0 ? this.noItems : undefined}
             .skeleton=${this.skeleton}
+            .separator=${this.separator}
           >
           </sonic-list>
         `;
